model-optimizer/mo/middle/passes/fusing/decomposition.py

```diff
@@ -39,8 +39,8 @@ def convert_batch_norm(graph: Graph):
     """
     nodes = graph.get_op_nodes()
     for node in nodes:
-        if node.has_valid('op') and (node.op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3',
```

Comment:
What is the difference between `model-optimizer/extensions/middle/FusedBatchNormNonConstant.py` and this function? Can we use just one of them?

Reply:
fixed